config/routes.rb

```diff
@@ -146,6 +146,7 @@
         end
         patch "suspend", on: :member
         patch "unsuspend", on: :member
+        patch "unexpire", on: :member
```

Comment:
I don’t love routes like these (I’d prefer to have a resource called Expiration, and DELETE it to unexpire in this instance), but given that it fits the existing pattern, I think I’m fine with it here.